### PR TITLE
Display rocket branch

### PR DESCRIPTION
**Pull request by soesandarwin2201**
## What does this PR do?
This PR is raised to create the Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation button or cancel the previously made booking

## Description of the task to be completed
- [x]  render fetch data from the SpaceX API endpoints
- [x]   the data are fetched, dispatch an action to store the selected data in Redux store
- [x] When a user clicks the "Reserve rocket" button make action needs to be dispatched to update the store
## How should it be tested
Clone the repository
```
git clone  https://github.com/shella12/React-group-project
```
Navigate to the folder
```
cd React-group-project
```
Checkout the branch
```
git checkout display-rocket-branch
```
#### Thanks for reviewing my PR :blush:

### CODE REVIEW

File: src/pages/Rocket/Rocket.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { bookRockets, cancelBooking } from '../../redux/Rockets/Rockets';
+
+const Rocket = (props) => {
+  const rocket = props;
+  const dispatch = useDispatch();
+
+  const bookRocket = (id) => {
+    console.log(id);
```
Comment (shella12) [inline src/pages/Rocket/Rocket.jsx:10]: - [x] [OPTIONAL] I suggest you remove console statements in your code. The purpose of console statements is debugging. After you are done debugging your code, it's best practice to remove any dead code from production code. Putting a console statement in your code may reduce the performance of the program

File: src/pages/Rocket/Rocket.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    dispatch(bookRockets(id));
+  };
+
+  const cancelRocket = (id) => {
+    console.log(id);
```
Comment (shella12) [inline src/pages/Rocket/Rocket.jsx:15]: -  [ ] [OPTIONAL] I suggest you remove console statements in your code. The purpose of console statements is debugging. After you are done debugging your code, it's best practice to remove any dead code from production code. Putting a console statement in your code may reduce the performance of the program

**Review by shella12 — APPROVED** [review]
Hi @soesandarwin2201 , 

## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Implemented all requirements 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

File: src/pages/Rocket/Rocket.jsx (hunk 2 above)
Comment (soesandarwin2201) [reply src/pages/Rocket/Rocket.jsx:15]: @shella12 thanks for your suggestion , i will make sure to remove console.log for later project .thanks for your time and effort.

**Pull request merged**